docker-compose.yml

```diff
@@ -37,71 +37,71 @@ services:
             - agency-network
         tty: true
 
-    sa-mc-controller:
-        environment:
-            - MULTI_AGENT=false
-            - MULTI_CONTROLLER=true
-            - PORT=3031
-            - SELF_URL=http://sa-mc-controller:3031
-        build:
-            context: .
-            dockerfile: Dockerfile
-        command: npm run start:debug
-        image: sa-mc-controller
-        container_name: sa-mc-controller
-        working_dir: /www
-        ports:
-            - "3031:3031"
-        volumes:
-            - ./:/www
-        networks:
-            - agency-network
-        tty: true
+    # sa-mc-controller:
```

Comment:
would it be easier to have a different docker-compose file for the configuration rather than uncomment/commenting sections (and risk checking that in)?

Reply:
Oops, didn't mean to commit that (and didn't even use the docker compose cause I figured out how to do a local test)